src/confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -670,7 +867,7 @@ def get_versions(self, subject_name):
 
         return self._rest_client.get('subjects/{}/versions'.format(_urlencode(subject_name)))
 
-    def delete_version(self, subject_name, version):
+    def delete_version(self, subject_name: str, version: int) -> int:
```

Comment:
There's no cache invalidation for delete version?

Reply:
Not currently, I can take a look if it's easy to add